Cargo.toml

```diff
@@ -43,3 +43,4 @@ v1 = []
 v3 = ["md5", "rand"]
 v4 = ["rand"]
 v5 = ["sha1", "rand"]
+const = ["nightly"]
```

Comment:
I would prefer this to be named `const_fn` as `const` could be constants or `const_fn`